pmd-plsql/etc/grammar/PldocAST.jjt

```diff
@@ -3379,7 +3467,7 @@ ASTPrimaryPrefix PrimaryPrefix() :
 | LOOKAHEAD(Literal()) ( simpleNode = Literal() ) { sb.append(simpleNode.getImage()) ; }
 | LOOKAHEAD(SimpleExpression()) ( simpleNode = SimpleExpression() ) { sb.append(simpleNode.getImage()); }
 | ( simpleNode =Name() ) { sb.append(simpleNode.getImage()) ; }
-| SelectStatement()
+// | SelectStatement()
```

Comment:
Is this an intended change? This looks like we won't be able to parse Select Statements correctly anymore (only within a subexpression, see next line).

Reply:
Yes, this was intended, because it smelled like a bug to me.
It probably was always meant as a comment, in particular if you look at the line immediately following this one, which you inserted 2 years ago.
A select statement is either a statement or (if used as an expression) it has to be enclosed in parentheses.
something like `varname := select sysdate from dual;` does not work and should not parse.